files/zh-cn/web/javascript/closures/index.html

```diff
@@ -12,7 +12,7 @@
 ---
 <div>{{jsSidebar("Intermediate")}}</div>
 
-<p>一个函数和对其周围状态（<strong>lexical environment，词法环境</strong>）的引用捆绑在一起（或者说函数被引用包围），这样的组合就是<strong>闭包</strong>（<strong>closure</strong>）。也就是说，闭包让你可以在一个内层函数中访问到其外层函数的作用域。在 JavaScript 中，每当创建一个函数，闭包就会在函数创建的同时被创建出来。</p>
+<p>我们将一个函数周边环境的状态称为<strong>词法环境</strong>（the <strong>lexical environment</strong>），那么当某个函数与其词法环境的引用捆绑在一起的时候，我们称这个组合为<strong>闭包</strong>（<strong>closure</strong>）。换而言之，闭包让开发者可以从内部函数访问外部函数的作用域。在 JavaScript 中，闭包会随着函数的每次创建同时被创建。</p>
```

Comment:
因为这一文档是主要描述闭包的，且第一段会作为搜索引擎检索的摘要，应保持尽量的简介明了：他人通过搜索引擎的摘要就能大致明白**什么是闭包**。现有的描述没法突出什么是“闭包”。以下是我的修改建议：

```suggestion
<p><strong>闭包</strong>（closure）是一个函数以及其捆绑的周边环境状态（<strong>lexical environment</strong>，<strong>词法环境</strong>）的引用的组合。换而言之，闭包让开发者可以从内部函数访问外部函数的作用域。在 JavaScript 中，闭包会随着函数的创建而被同时创建。</p>
```